monkey/monkey_island/cc/services/attack/technique_reports/attack_technique_test.py

```diff
@@ -0,0 +1,98 @@
+import copy
```

Comment:
`attack_technique_test.py` should be renamed to `test_attack_technique.py`. `test` needs to be prepended, because unit test will not find it otherwise and won't run it automatically with other tests.

Reply:
Done